coding-articles/Article.md

```diff
@@ -0,0 +1,584 @@
+```
+mkdir deposit-detector
+cd deposit-detector
+npx forta-agent@latest init --typescript
+```
+> ⚠ please use npm package manager for installation not yarn because you will face some issues while deploying the bot.
+
+The above command will install a nodejs project with the following packages:-
+- forta-agent:  The most important one is to use the bot.
+- forta-agent-tools:  For testing and network management. (I will install it later)
+- jest:  For testing.
+- nodemon:  For the dev server.
+- typescript,@types/jest,@types/nodemo,ts-jest:  To use Typescript
+
+After installation, you should find something like that.
+
+![project-root-files](./images/project-root.png)
+
+That's a ready template of a working bot. All you need to do is run `npm start` And the bot will start emitting findings in all transfer events on the [Tether](https://etherscan.io/address/0xdac17f958d2ee523a2206206994597c13d831ec7) token deployed on the Ethereum blockchain. How to do this?
+
+## How does the bot listen to transactions and blocks?
+To answer this question, go to `src/agent.ts` , or you can check the file content [here](https://github.com/forta-network/forta-bot-sdk/blob/master/starter-project/ts/src/agent.ts#L65). It is the main file of the bot where you can add your bot logic. You will find two exports functions
+
+```
+export default {
+handleTransaction,
+// handleBlock
+};
+```
+**handleTransaction:** Is called each time a new transaction has been submitted. handleTransaction receivesTransactionEvent an object which contains all events that occurred inside a transaction. You can check all TransactionEvent properties from the [documentation](https://docs.forta.network/en/latest/sdk/#transactionevent).
+
+**handleBlock:** Is called each time a new block is minted. handleBlock receives BlockEvent an object which contains all transactions that occurred inside a block. You can check all BlockEvent properties from the [documentation](https://docs.forta.network/en/latest/sdk/#blockevent).
+
+I need the bot to detect deposits inside the "Deposit.sol" smart contract. So I will use only the `handleTransaction`.
+
+## Build The Deposit Detector
+
+Before going to the code, let's think about the steps to build a bot:
+1. Blockchain Network: Which blockchain networks do we want to use?
+2. Network Manager: Specify the smart contract addresses for different networks we want to monitor.
+3. Emit Findings(send notification): Decide which events we want to listen to. 
+
+    In our case, it is only Deposited event:
+
+      - If the Deposited event is called.
+      - If the deposited amount is less than 0.01
+      - If the deposited address is blacklisted.
+
+## Configure Your Bot To Different Blockchain Networks
+In my case, I will add Mumbai Testnet network. And show you how to add multiple ones.
+
+
+To do this, I will edit two files.
+
+
+First file `package.json` Replace "1" Ethereum Mainnet chain Id with "8001" Mumbai network chain Id
```

Comment:
The chain id for mumbai is 80001, not 8001. I think this is a typo, since you filled the right value below.

Reply:
yes fixed